invenio_query_parser/ast.py

```diff
@@ -182,3 +186,7 @@ class RegexValue(Leaf):
 
 class EmptyQuery(Leaf):
     pass
+
+
+class MalformedQuery(Leaf):
```

Comment:
Ideally yes. This is then needed to implement: https://github.com/inspirehep/inspire-next/pull/1182